juniper_official/System/check-vmcores.rule

```diff
@@ -0,0 +1,43 @@
+/*
+ * This rule detects presence of vmcore (Kernel crash) file(s) on the device.
+ * If vmcore is detected, this rule raises a yellow alarm. It will continue
+ * to report yellow alarm till the vmcore is moved from /var/crash into
+ * some other location or some other device.
+ */
+iceberg {
+    topic system {
+        synopsis "Check presence of Kernel vmcore files";
+        description "This rule detects presence of Kernel vmcore files";
+        rule check-vmcores {
+            sensor vmcores-sensor {
```

Comment:
Provide sensor description and synopsis.

Reply:
Addressed.